conanfile.py

```diff
@@ -184,6 +192,9 @@ def requirements(self):
             ]
             for lib in systemLibsOverrides:
                 self.requires(f"{lib}@vcmi/apple", override=True)
+        elif self.settings.os == "Windows":
```

Comment:
no need to specify explicit OS here:
```suggestion
        else:
```

Reply:
done